_component_design/buttons-iconography.md

```diff
@@ -0,0 +1,354 @@
+---
+title: Button Icons
+parent: Inputs and Controls
+layout: component
+category: Components
+usage: >
+  An icon button enables a user to perform an action or initiate a process on a Control Panel page. A button directs the user's attention to important actions.
+preview-image: preview-images/button-icon_thumbnail.svg
+status: stable
+resource: true
+last-modified: 2018-09-28
+helix-ui-css: true
+helix-ui-javascript: false
+---
+
+{% include toc.html %}
+
+<section class="static-section" markdown="1">
+
+<div class="hxRow" markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
+
+## When to use
+
+Add an icon to a button when you want to visually represent an action to the user.
+
+{% endcolumn %}
+
+</div>
+
+</section>
+
+<section class="static-section" markdown="1">
+
+<div class="hxRow" markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
+
+## Best practices
+
+Consider the following best practices when you construct a button that includes an icon:
+
+- Do not add a icon to a button unless it is immediately recognizable to the user. Icons are often culturally defined. For example, shopping carts don’t exist in all areas of the world.
```

Comment:
maybe we should just rephrase the NN example?

from NN:
"For example, mailboxes look very different in various countries whereas envelopes look the same, therefore an envelope is a more international icon for an email program than a mailbox"

Reply:
tweaked your wording, and yes, using the example you provided. love. it.